src/main/scala/com/gu/invoicing/refundErroneousPayment/Impl.scala

```diff
@@ -52,30 +49,39 @@ object Impl {
       .Status
   }
 
-  def getBalancingInvoice(accountId: String, invoiceAmount:BigDecimal):String =
+  def getInvoices(accountId: String): List[Invoice] =
     Http(s"$zuoraApiHost/v1/action/query")
       .header("Authorization", s"Bearer $accessToken")
       .header("Content-Type", "application/json")
-      .postData(s"""{"queryString": "SELECT Id, Amount, Balance, InvoiceDate, InvoiceNumber, PaymentAmount, TargetDate, Status FROM Invoice WHERE AccountId = '$accountId' AND status = 'Posted'"}""")
+      .postData(
+        s"""{"queryString": "select Id, Amount, Balance, InvoiceDate, InvoiceNumber, PaymentAmount, TargetDate, Status from Invoice where AccountId = '$accountId'"}"""
```

Comment:
why not Posted any more?

Reply:
I copied over the old query.  Reinstated the extra filter now.